apps/barista/src/components/live-example/live-example.ts

```diff
@@ -126,18 +150,49 @@ export class BaLiveExample implements OnDestroy {
     if (this._componentRef) {
       this._componentRef.destroy();
     }
+    this._timerSubscription.unsubscribe();
   }
 
-  /** Whether one of the three sources has been set. */
+  /** @internal Whether one of the three sources has been set. */
   _hasSources(): boolean {
     return Boolean(this.classSource || this.templateSource);
   }
 
+  /** @internal Sets the active tab. */
   _setActiveTab(tab: 'html' | 'ts' | 'scss'): void {
     this._activeTab = tab;
     this._activeTabChanged = true;
   }
 
+  /** @internal Copies content of currently active tab to clipboard. */
+  _copyToClipboard(): void {
+    if (document) {
```

Comment:
make a platform check with cdks platform library in case that universal or jest has a document as an empty object where there is no execCommand on it.